src/main/java/com/purbon/kafka/topology/actions/access/builders/BuildBindingsForKStreams.java

```diff
@@ -35,7 +35,11 @@ protected void execute() throws IOException {
 
     bindings =
         builderProvider.buildBindingsForStreamsApp(
-            app.getPrincipal(), prefix, readTopics, writeTopics);
+            app.getPrincipal(),
+            prefix,
+            readTopics,
+            writeTopics,
+            app.getExactlyOnce().orElse(false));
```

Comment:
Thinking out loud, can we have this default in the model? what do you think?

Reply:
Agree. Will initialize eos on the constructors, and `get()` here instead of `orElse()`. Let me know how it looks like now.

Reply:
❤️  it, thanks a lot man! appreciate it